src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -218,10 +219,11 @@ export const RuntimeEventDetails: FC<{
           <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
             <dt>{t('runtimeEvent.fields.owner')}</dt>
             <dd>
-              <AccountLink address={event.body.owner} scope={scope} />
-              {addressSwitchOption === AddressSwitchOption.Oasis && (
-                <CopyToClipboard value={event.body.owner} />
-              )}
+              <AccountLinkWithAddressSwitch
```

Comment:
RuntimeAccountLink and type vs addressSwitchOption?  at least for me it would be more readable